### PR TITLE
Rename the nullable text input interface option

### PR DESCRIPTION
**Pull request by nickrum**
The "nullable" option is kind of confusing because the phrase "Save empty value as NULL" implies that an empty value is _always_ saved as `NULL`, although the option only affects the value of a text field if it's content has been removed manually. Also, it is not clear, that the alternative to `NULL` is in fact the empty string.

I am not too happy with the new wording either, so I would love to hear your thoughts/ideas.

### CODE REVIEW

**Comment by benhaynes** [comment]
This seems like a good change because if the Column does not have "Allow NULL" then, this interface option won't make sense anyway. However it seems a bit more confusing to say "Save cleared value as empty string", since I would assume a cleared value is saved as an empty string. Saying NULL makes a little more sense to me in that respect.

Also, if we do change the option text, let's make sure it fixes without wrapping/ellipses.

Screen Shot 2021-04-14 at 10 12 49 AM

**Comment by rijkvanzanten** [comment]
@benhaynes final thoughts on the wording here?

**Comment by benhaynes** [comment]
How about this?

Screen Shot 2021-04-14 at 11 15 51 AM

**Comment by nickrum** [comment]
> However it seems a bit more confusing to say "Save cleared value as empty string", since I would assume a cleared value is saved as an empty string.

Given `NULL` is the default value of the "Default Value" option for string fields, I would actually assume the opposite 😅

As to a user there is no visual difference between a text field that hasn't been written to and a text field that had content and was manually deleted by the user, so there shouldn't be a difference in behavior as well. The option essentially only serves as an escape hatch for the more sane default behavior.

**Comment by benhaynes** [comment]
That makes sense, and I agree! Glad this will help us be consistent regardless of initial or edited/cleared values.

File: app/src/lang/translations/en-US.yaml [hunk 1]
```diff
@@ -994,8 +994,8 @@ interfaces:
     trim_label: Trim the start and end
     mask: Masked
     mask_label: Hide the real value
-    nullable: Nullable
-    nullable_label: Save empty value as NULL
+    clear: Clear
```
Comment (rijkvanzanten) [inline app/src/lang/translations/en-US.yaml:997]:
```suggestion
    clear: Cleared Value
```

File: app/src/lang/translations/en-US.yaml [hunk 2]
```diff
@@ -994,8 +994,8 @@ interfaces:
...
+    clear_label: Save cleared value as empty string
```
Comment (rijkvanzanten) [inline app/src/lang/translations/en-US.yaml:998]:
```suggestion
    clear_label: Save as empty string
```

**Review by rijkvanzanten — CHANGES_REQUESTED** [review]

**Review by rijkvanzanten — APPROVED** [review]

**Pull request merged**